src/pages/lessons/[slug].tsx

```diff
@@ -390,6 +390,10 @@ const Lesson: React.FC<LessonProps> = ({
             send('SUBSCRIBE')
           }
         }
+        videoService.send({
+          type: 'LOAD_RESOURCE',
+          resource: lessonState.context.lesson,
+        })
```

Comment:
after the actual video is loaded (not the basic SEO safe version) we need to pitch it into the video player